packages/notebook/src/common/notebook-common.ts

```diff
@@ -0,0 +1,348 @@
+// *****************************************************************************
+// Copyright (C) 2023 Red Hat, Inc. and others.
```

Comment:
A lot of files contain the Red Hat header. You can safely replace those with `TypeFox`.